src/main/java/com/hubspot/jinjava/el/ext/DivOperator.java

```diff
@@ -0,0 +1,43 @@
+package com.hubspot.jinjava.el.ext;
+
+import de.odysseus.el.misc.TypeConverter;
+import de.odysseus.el.tree.impl.Parser.ExtensionHandler;
+import de.odysseus.el.tree.impl.Parser.ExtensionPoint;
+import de.odysseus.el.tree.impl.Scanner;
+import de.odysseus.el.tree.impl.ast.AstBinary;
+import de.odysseus.el.tree.impl.ast.AstBinary.SimpleOperator;
+import de.odysseus.el.tree.impl.ast.AstNode;
+
+/**
+ * Created to allow for the detection and handling of divide-by-zero requests in EL expressions
+ * (see PR 473 @ https://github.com/HubSpot/jinjava/pull/473)
+ */
+public class DivOperator extends SimpleOperator {
+  public static final Scanner.ExtensionToken TOKEN = new Scanner.ExtensionToken("/");
+  public static final DivOperator OP = new DivOperator();
+
+  @Override
+  protected Object apply(TypeConverter converter, Object a, Object b) {
+    if (a == null || b == null) {
+      throw new IllegalArgumentException("Division argument may not be null");
```

Comment:
@bszabo, we should be throwing an `InvalidArgumentException` here instead of `IllegalArgumentException`. Sorry for not catching that earlier.

Reply:
I've made the change both here and lower down in the file, where detection of a zero divisor occurs